src/backends/tensorrt/plan_backend.h

```diff
@@ -98,12 +105,17 @@ class PlanBackend : public InferenceBackend {
     nvinfer1::IRuntime* runtime_;
     nvinfer1::ICudaEngine* engine_;
     nvinfer1::IExecutionContext* context_;
+    int profile_index_;
```

Comment:
Can you add a comment for these 4 items, perhaps put is_dynamic_ first and then say in general what these are used for. Are they only used for dynamic shapes, etc.